### ISSUE TITLE
Error with ! in password

### ISSUE THREAD

**Issue by kps1ny** [idrm/create-panono-backup#7]
Hi,

My first attempt to run the tool resulted in an error (via Mac Terminal):

-bash: !": event not found

It turns out that happened because my password had a ! character in it. (Found the answer via Google and Stack Overflow that provided a clue that the ! was being interpreted as a Terminal history reference.)

When I removed the password from the command line, and instead entered it via prompt, everything was fine.

So you may want to make a note that at least some password characters can fail when used via a command-line parameter.

-Ken

**Comment by idrm** [comment]
Have you tried enclosing the password in double quotes inside the command line parameters?

**Comment by kps1ny** [comment]
Yes, per the example. Apparently the ! is still interpreted as outside of them.

**Comment by kps1ny** [comment]
Tried it with a \ backslash to escape the ! as well. That avoids the OS error, but the Panono site then rejects the password.
